<ui version="4.0" >
 <class>PrefsLNFWidget</class>
 <widget class="QWidget" name="PrefsLNFWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>These features require application restart</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QComboBox" name="styleComboBox" >
        <property name="toolTip" >
         <string>Select the application look/GUI theme</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>GUI &amp;Language:</string>
        </property>
        <property name="buddy" >
         <cstring>languageComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="languageComboBox" >
        <property name="toolTip" >
         <string>Select the application interface language</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>GUI &amp;Style:</string>
        </property>
        <property name="buddy" >
         <cstring>styleComboBox</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>&amp;Recently Used Databases:</string>
       </property>
       <property name="buddy" >
        <cstring>recentlyUsedSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="recentlyUsedSpinBox" >
       <property name="toolTip" >
        <string>Count of the items in the Recent menu</string>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>304</width>
       <height>144</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="titleLabel" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Look and Feel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="openLastDBCheckBox" >
     <property name="toolTip" >
      <string>Remember the last used database for next application start</string>
     </property>
     <property name="text" >
      <string>Ope&amp;n Last Database on Start</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="openLastSqlFileCheckBox" >
     <property name="text" >
      <string>Open Last SQL Document on Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
